tools/install.sh

```diff
@@ -13,37 +26,42 @@ fi
 
 echo "--  Checking for application directory"
 applicationDir="${HOME}/.terra"
-mkdir -p $applicationDir
+mkdir -p "$applicationDir"
 if [ ! -d "${applicationDir}" ]; then
     echo "Error creating application directory: ${applicationDir}"
     exit 1
 fi
 
 echo "--  Moving JARs to application directory"
 if [ -f "${applicationDir}/lib" ] || [ -d "${applicationDir}/lib" ]; then
-  rm -R "${applicationDir}/lib"
+  rm -R "${applicationDir:?}/lib"
```

Comment:
I think this will print an error if `applicationDir` isn't set, but I see that variable is defined above. Did you see some error with this line?

Reply:
The IDE suggested it, so we don't delete `/lib` if somehow it becomes `null` under maintenance.